### DISCUSSION TITLE
insert hint rounded corners

### DISCUSSION THREAD

**Discussion by ThatOneCalculator** [YaLTeR/niri#768] [Ideas]
It'd be nice to have the option to enable rounded corners on insert hints to match window rounded corners

![image](https://github.com/user-attachments/assets/60afc072-2b5f-4ccc-972c-2b8b5bdf87ff)

![image](https://github.com/user-attachments/assets/b6f767cc-58b4-4391-b703-cc59473c8824)

**Comment by YaLTeR** [comment]
Hmm, yeah. Currently the insert hint uses a simple solid color element. It needs to use the full border render element, which will let it use rounded corners and also gradient fill.

**Comment by YaLTeR** [comment]
Implemented in fce5d668783b4bbd5a6cea5e816d5d90a810d8e6.
